content/en/blog/_posts/2018-12-11-Kubernetes-Federation-Evolution.md

```diff
@@ -82,8 +82,6 @@ For external clients, automatic DNS expansion described is not currently possibl
 
 That way, your clients can always use the short form on the left, and always be automatically routed to the closest healthy shard on their home continent. All of the required failover is handled for you automatically by Kubernetes cluster federation.
 
-As further reading, a more elaborate example for users is available in the [Multi-Cluster Service DNS with ExternalDNS guide](https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/servicedns-with-externaldns.md).
-
```

Comment:
Rather than remove the paragraph, we can link to that doc: https://github.com/kubernetes-retired/kubefed/blob/dbcd4da3823a7ba8ac29e80c9d5b968868638d28/docs/servicedns-with-externaldns.md

Reply:
> Rather than remove the paragraph, we can link to that doc: https://github.com/kubernetes-retired/kubefed/blob/dbcd4da3823a7ba8ac29e80c9d5b968868638d28/docs/servicedns-with-externaldns.md

@sftim  updated with the suggested link, please review it.